examples/hello-world/src/main/kotlin/com/justai/jaicf/examples/helloworld/channel/viber/scenario/ViberTestScenario.kt

```diff
@@ -0,0 +1,277 @@
+package com.justai.jaicf.examples.helloworld.channel.viber.scenario
+
+import com.justai.jaicf.BotEngine
+import com.justai.jaicf.activator.regex.RegexActivator
+import com.justai.jaicf.builder.Scenario
+import com.justai.jaicf.channel.viber.api.CarouselElement
+import com.justai.jaicf.channel.viber.api.ViberEvent
+import com.justai.jaicf.channel.viber.api.isReceivedAsSilent
+import com.justai.jaicf.channel.viber.contact
+import com.justai.jaicf.channel.viber.conversationStarted
+import com.justai.jaicf.channel.viber.delivered
+import com.justai.jaicf.channel.viber.file
+import com.justai.jaicf.channel.viber.location
+import com.justai.jaicf.channel.viber.picture
+import com.justai.jaicf.channel.viber.sdk.api.KeyboardBuilder
+import com.justai.jaicf.channel.viber.sdk.api.OpenMapButton
+import com.justai.jaicf.channel.viber.sdk.api.RedirectButton
+import com.justai.jaicf.channel.viber.sdk.api.ReplyButton
+import com.justai.jaicf.channel.viber.sdk.api.ViberButton
+import com.justai.jaicf.channel.viber.sdk.message.Location
+import com.justai.jaicf.channel.viber.sdk.message.PictureMessage
+import com.justai.jaicf.channel.viber.seen
+import com.justai.jaicf.channel.viber.sticker
+import com.justai.jaicf.channel.viber.subscribed
+import com.justai.jaicf.channel.viber.text
+import com.justai.jaicf.channel.viber.viber
+import com.justai.jaicf.channel.viber.video
+import java.io.File
+
+val ViberTestScenario = Scenario {
```

Comment:
I don't think this scenario should exist in Hello-world example. But the code is useful, so I'd suggest moving it to separate module, like `viber-sdk-example`.

Reply:
You're right, I created an additional package for this. I also added a config class that can be used to specify to ignore seen and delivered events, so the viber channel can now be used in the default HelloWorldScenario.